integration-tests/ws-security-policy/src/main/resources/saml1-policy.xml

```diff
@@ -5,33 +5,11 @@
         xmlns:sp="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702">
     <wsp:ExactlyOne>
         <wsp:All>
-            <sp:TransportBinding>
-                <wsp:Policy>
-                    <sp:TransportToken>
-                        <wsp:Policy>
-                            <sp:HttpsToken>
-                                <wsp:Policy/>
-                            </sp:HttpsToken>
-                        </wsp:Policy>
-                    </sp:TransportToken>
-                    <sp:Layout>
-                        <wsp:Policy>
-                            <sp:Lax/>
-                        </wsp:Policy>
-                    </sp:Layout>
-                    <sp:IncludeTimestamp/>
-                    <sp:AlgorithmSuite>
-                        <wsp:Policy>
-                            <sp:Basic128/>
-                        </wsp:Policy>
-                    </sp:AlgorithmSuite>
-                </wsp:Policy>
-            </sp:TransportBinding>
```

Comment:
TransportBinding, AsymmetricBinding and SymmetricBinding are security bindings, they are different ways to protect the message.

TransportBinding is the transport level(HTTPS) to protect message, while AsymmetricBinding and SymmetricBinding are mesasge encoding ways to protect the message. 

As in quarkus applications, we configure the HTTPS at platform vertx sevlet level(in the application.properties), so here in the ws-secuirtypolicy xml, we shouldn't have TransportBinding. Plus, I believe in CXF we only support one security binding in  ws-secuirtypolicy xml, it should be either TransportBinding, AsymmetricBinding or SymmetricBinding. Though I can't find concrete definition of this in ws-secuirtypolicy specification, I make this conclusion by reading the related code in CXF,  also I found this from
https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/6.3/html-single/apache_cxf_security_guide/index
```
As described in the WS-SecurityPolicy specification, one of the following binding types can be used to protect SOAP messages: 
```

So this means TansportBinding and AsymmetricBinding shouldn't co-exist, this is the reason that in stream mode the AsymmetricBinding is ignored because CXF picks up TansportBinding. While in Dom mode it works because CXF picks AsymmetricBinding precede the TansportBinding. So here the behaviour is different between Dom and Stream mode, but only one Security Binding(TransportBinding, AsymmetricBinding or SymmetricBinding) will be used in both mode.